src/mono/wasm/Wasm.Build.Tests/BuildPublishTests.cs

```diff
@@ -73,7 +73,7 @@ public void BuildThenPublishNoAOT(BuildArgs buildArgs, RunHost host, string id)
         [BuildAndRun(host: RunHost.Chrome, aot: true, config: "Debug")]
         public void BuildThenPublishWithAOT(BuildArgs buildArgs, RunHost host, string id)
         {
-            string projectName = $"build_publish_{buildArgs.Config}";
+            string projectName = $"build_publish_{buildArgs.Config}_{s_unicodeChar}";
```

Comment:
Maybe add a comment explaining why s_unicodeChar being here is important

Reply:
Thank you, added (in the common function for creating the test project name).